deployment/clouddeploy/gke-workers/base/combine-to-osv.yaml

```diff
@@ -4,11 +4,11 @@ metadata:
   name: combine-to-osv
 spec:
   schedule: "30 */1 * * *"
-  concurrencyPolicy: Forbid
+  concurrencyPolicy: Allow # on the assumption the dominant part of the runtime is the GCS copy at the end
```

Comment:
Is there a risk we'll end up with two jobs trying to do the GCS copy at the same time (and would that be a problem)?

Reply:
> Is there a risk we'll end up with two jobs trying to do the GCS copy at the same time (and would that be a problem)?

Yes there is, and I'm of the opinion that it won't be a problem because:

- the source files being copied will be in discretely separate pods
- the concurrent jobs will be operating at different points in these discrete sets of files

i.e. newer job B will start at (expected different) file 1 (of ~39K) while older job B is already up to file 22K (of ~39K) and the set of files that actually require transferring (as opposed to getting their modification time updated) will be disjoint.

I'm happy to be proven wrong here.

Reply:
Sounds reasonable to me